assessment/collectors/app_migration_analysis.go

```diff
@@ -524,9 +534,38 @@ func (m *MigrationSummarizer) AnalyzeProject(ctx context.Context) (*CodeAssessme
 			}
 		}
 	}
+	codeAssessment.Language = language
+	codeAssessment.Framework = framework
+	codeAssessment.TotalLoc = totalLoc
+	codeAssessment.TotalFiles = fileIndex
 	return codeAssessment, nil
 }
 
+func getFramework(fileContent string) string {
```

Comment:
This method is specific to Go ecosystem, we should also update it as part of refactoring to support multiple languages and frameworks.